Luminary099/R31.agc

```diff
@@ -99,7 +99,7 @@ DOCMBASE	STORE	BASETIME	# PRECISION BASE VECTOR FOR CM
 REV83		CS	FLAGWRD7
 		MASK	AVEGFBIT
 		EXTEND
-		BZF	GETRVN		# IF AVEGFLAG SET, USE RN,VN
+		BZF	GETRVN		# IF AVEGFLAG SET, USE RN.VN
```

Comment:
This is a comma

![image](https://user-images.githubusercontent.com/3440094/95020453-2196ec00-0663-11eb-8f42-b1ed6b35f29e.png)


```suggestion
		BZF	GETRVN		# IF AVEGFLAG SET, USE RN,VN
```